tests/integration/server/roles_api_test.py

```diff
@@ -48,3 +71,27 @@ def test_create_duplicate_role():
         assert response.json()["code"] == 400
         delete_user_by_username("susan22")
         delete_role_by_name("Manager1")
+
+
+def test_syncdirectionflag_rolename():
+    """ Testing the presence and the value of syncdirection flag  is set to OUTBOUND of a role in roles table. """
+    new_rolename = "ManagerRandom20"
+    new_username = "susansonrandom20"
```

Comment:
The role and user needs to be deleted after this test has finished with assertions.

Reply:
![image](https://user-images.githubusercontent.com/23491817/55590659-59c51380-56e8-11e9-9828-e1e7a252c05e.png)
Already deleting the created user & roles.